src/org/parosproxy/paros/db/DbUtils.java

```diff
@@ -275,4 +296,21 @@ public static void executeUpdateAndClose(PreparedStatement preparedStatement) th
         }
     }
 
+
+    /**
+     * Utility function to execute update the given {@code sql}

```

Comment:
The wording here is a little awkward. Do we need a separate executor for update statements? Maybe something like:
`Utility function to execute the given {@code sql} update statement`
or something like that...

Although now that I [read](https://javaconceptoftheday.com/difference-between-executequery-executeupdate-execute-in-jdbc/) about it, it could be `update`, `insert`, or `delete`... So perhaps `...statement that updates the DB in some way`